src/test/java/org/jabref/logic/l10n/LocalizationConsistencyTest.java

```diff
@@ -117,6 +117,24 @@ public void keyValueShouldBeEqualForEnglishPropertiesMessages() {
     }
 
     @Test
+    public void languageKeysShouldNotBeQuotedInFiles() throws IOException {
+        final List<LocalizationEntry> quotedEntries = LocalizationParser
+                .findLocalizationParametersStringsInJavaFiles(LocalizationBundleForTest.LANG)
+                .stream()
+                .filter(key -> key.getKey().contains("_") && key.getKey().equals(new LocalizationKey(key.getKey()).getPropertiesKey()))
```

Comment:
I wonder if `.contains('_')` will produce false positives one day. Probably not very likely, as there are no problem so far.

Reply:
@LinusDietz We need *some* test or we will get a lot of false positives for messages that consist only of a single word like "Cancel". I didn't put too much thought into this. It was the first that came to mind.

Reply:
@halirutan I agree with the approach and don't have any other reasonable idea to solve this. My remark is entirely serves karma purposes, to have a 'told you so moment' if somebody actually needs to put an underscore into a localized String (;